src/app/navigation/AppNavigation.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {createDrawerNavigator} from '@react-navigation/drawer';
```

Comment:
Wdyt if we look for an eslint rule that allows us to have space between the `{ }`?

For example,
 ```suggestion
import { createDrawerNavigator } from '@react-navigation/drawer';
```

Reply:
we can add it on prettierrc :
`bracketSpacing: true`